Append to CFLAGS and CPPFLAGS rather than overwriting them.

See: https://github.com/WillEccles/cfm/pull/15

Honor LDFLAGS when linking.

See: https://github.com/WillEccles/cfm/pull/16
--- Makefile.orig	2020-02-27 13:06:22.000000000 -0600
+++ Makefile	2020-05-15 07:03:47.000000000 -0500
@@ -5,15 +5,15 @@
 MANPAGE = cfm.1
 PREFIX ?= /usr/local
 
-CFLAGS = -O3 -std=c11 -Wall -W -pedantic
-CPPFLAGS = -D_XOPEN_SOURCE=700
+CFLAGS += -O3 -std=c11 -Wall -W -pedantic
+CPPFLAGS += -D_XOPEN_SOURCE=700
 
 .PHONY: install uninstall clean
 
 all: $(TARGET)
 
 $(TARGET): $(CONF) $(SRC)
-	$(CC) $(CFLAGS) $(CPPFLAGS) $(SRC) -o $@
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) $(SRC) -o $@
 
 $(CONF):
 	@cp -v $(DEFCONF) $(CONF)
